src/main/git-elegant-check

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
+default() {
+    case "$1" in
```

Comment:
@extsoft Then is there any sense in "-a" and "--all" flags if we don't check it anywhere?

Reply:
@extsoft There is a question about your comment

> And code duplication:
git diff --check is used twice
git diff --cached --check is used twice

Function can be changed to:

```
default() {
    case "$1" in
        "-u" | "--unstaged")
            git diff --check
            ;;
        "-s" | "--staged")
            git diff --cached --check
            ;;
        *)
            git-elegant check -u
            git-elegant check -s
            ;;
    esac
```
But in my opinion calling `git-elegant check -u` or `git-elegant check -s` consumes system resources. Is it reasonable to use them instead of direct commands to git?

Reply:
@ivan-podorozhnyi 

> But in my opinion calling git-elegant check -u or git-elegant check -s consumes system resources

How? Could you give some insights?

Reply:
@extsoft In my opinion calling `git` commands directly consume less system resources than `git-elegant` commands. This is because `git-elegant` commands pass additional checks and call other commands. Maybe its not that significant load for git-elegant app. 
Do you think that `check` function  in my previous message is better than current in branch `feature/git-elegant-check`?

Reply:
@ivan-podorozhnyi I see. Leave it for now as it's implemented. I don't think we need some performance optimisations now.